tests/kernel/fatal/exception/src/main.c

```diff
@@ -79,6 +79,9 @@ void entry_cpu_exception(void *p1, void *p2, void *p3)
 	__asm__ volatile ("trap");
 #elif defined(CONFIG_ARC)
 	__asm__ volatile ("swi");
+#elif defined(CONFIG_RISCV) && defined(CONFIG_XIP)
+	/* illegal instruction on RISCV */
+	__asm__ volatile (".long 0");
```

Comment:
It seems this is happily ignored by renode simulator for hifive1... maybe use another instruction, like `.word 0xffffffff` (like the `0x77777777` below)?
And I guess you then don't need the `CONFIG_XIP` check.

Reply:
Done